planner/core/planbuilder.go

```diff
@@ -2522,7 +2522,7 @@ func (b *PlanBuilder) buildAnalyzeAllIndex(as *ast.AnalyzeTableStmt, opts map[as
 }
 
 // CMSketchSizeLimit indicates the size limit of CMSketch.
-var CMSketchSizeLimit = kv.TxnEntrySizeLimit / binary.MaxVarintLen32
+var CMSketchSizeLimit = kv.TxnEntrySizeLimit.Load() / binary.MaxVarintLen32
```

Comment:
This variable is initialized once and will not be updated with the change of `TxnEntrySizeLimit`, it could be a risk.

@XuHuaiyu @chrysan  Do you have any idea how this should work with the new `tidb_entry_size_limit` variable?